src/ex.rs

```diff
@@ -167,6 +175,27 @@ pub fn define_(
 }
 
 impl Experiment {
+    pub fn validate(&self) -> Result<()> {
+        if self.toolchains[0] == self.toolchains[1] {
+            bail!("reusing the same toolchain isn't supported");
+        }
```

Comment:
I assume it wouldn't be much effort to support this e.g. if you wanted to compare with and without a rustflag? Maybe just removing this check?

Reply:
(I should say - this is idle curiosity, not a suggestion that we should support this now)

Reply:
Well, this is actually supported right now! `stable` and `stable+rustflags` are considered by Crater two different toolchains, the check only prevents comparing `stable` and `stable`.